src/cdk/v2/destinations/intercom/procWorkflow.yaml

```diff
@@ -77,8 +77,8 @@ steps:
     template: |
       const payload = .message.context.mappedToDestination ? $.outputs.rEtlPayload : $.outputs.identifyTransformationForLatestVersion;
       payload.name = $.getName(.message);
-      payload.custom_attributes = .message.context.traits || {};
-      payload.custom_attributes = $.filterCustomAttributes(payload, "user", .destination);
+      payload.custom_attributes = !.message.context.mappedToDestination ? .message.context.traits || {} : .message.traits.custom_attributes || {};
```

Comment:
payload.custom_attributes = (.message.context.mappedToDestination ? .message.traits.custom_attributes : .message.context.traits) || {};

Reply:
can be rewritten like this